community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/spi/QueriedGraphStatistics.scala

```diff
@@ -102,10 +102,31 @@ class QueriedGraphStatistics(graph: GraphDatabaseService, queryContext: QueryCon
       if (values.isEmpty)
         Some(Selectivity(0)) // Avoids division by zero
       else
+        // TODO: Review this, because this does not match the equation in TransactionBoundGraphStatistics
         Some(Selectivity(1.0 / values.size))
     }
   }
 
+  def indexPropertyExistsSelectivity(labelId: LabelId, propertyKeyId: PropertyKeyId): Option[Selectivity] = {
+    // TODO: This class appears to only be used by tests. Determine the point of this class and whether we need a different implementation below.
```

Comment:
It was added before the statistics was in place in order to test planning if I remember correctly